charts/adoption-backend/Chart.yaml

```diff
@@ -3,6 +3,6 @@ appVersion: "1.0"
 description: A Helm chart for adoption-backend-service app
 name: adoption-backend
 home: https://github.com/hmcts/adoption-backend
-version: 0.0.1
+version: 1.3.0
```

Comment:
why such big jump, should not be 0.0.2 ?

Reply:
updated